pkgs/development/python-modules/invocations/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, blessings
+, fetchFromGitHub
+, invoke
+, lib
+, releases
+, semantic-version
+, tabulate
+, twine
+}:
+
+buildPythonPackage rec {
+  pname = "invocations";
+  version = "2.6.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "pyinvoke";
+    repo = pname;
+    rev = version;
+    hash = "sha256-eyOJKVRfn7elyEkERl7hvRTNFmq7O9Pr03lBS6xp0wE=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "semantic_version>=2.4,<2.7" "semantic_version" \
+      --replace "tabulate==0.7.5" "tabulate"
+  '';
+
+  propagatedBuildInputs = [ blessings invoke releases semantic-version tabulate twine ];
+
+  # There's an error loading the test suite: https://gist.github.com/samuela/ade41eb30b3029b5f4b706d7c012b2bd.
+  doCheck = false;
```

Comment:
Can you try an older pytest version?

Reply:
I wasn't aware that was a possibility? How do I run with an older pytest version?

Reply:
I'm only seeing one version in the current tree: https://github.com/nixos/nixpkgs/tree/master/pkgs/development/python-modules/pytest.

Reply:
Well, no longer since https://github.com/NixOS/nixpkgs/pull/158848/commits which dropped some older pytests...